DEVELOPERS.md

```diff
@@ -249,3 +249,34 @@ Hint: To find both the declaration and definition of a function named
 ```bash
 git grep -nWG "^[^[:blank:]].*func_name("
 ```
+
+## JSON
```

Comment:
Awesome idea! I would add also the final JSON printed by the following code, just to be sure.

Reply:
Added.